src/javascript/Edit/index.js

```diff
@@ -34,53 +34,62 @@ export const registerActions = actionsRegistry => {
         target: ['editHeaderActions:1']
     });
 
-    actionsRegistry.add('startWorkflowMainButton', startWorkflowMainButton, {
+    actionsRegistry.add('startWorkflowMainButton', startWorkflow(true), {
```

Comment:
instead of passing true to the component, you can just set a new property:

isMainButton: true
inside the action, like the buttonLabel, buttonIcon, target, etc ...

it will be passed to the action context, and you can reuse it directly in the action code using:

context.isMainButton